### PR TITLE
Updating missing data for risk analysis

### PR DESCRIPTION
**Pull request by deepsm007**
Based on requirements from risk analysis, I have added a block in the test step

/cc @neisw

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh [hunk 1]
```diff
@@ -417,6 +417,45 @@ do
 done
 echo "$(date) - all imagestreams are imported."
 
+echo "Collecting cluster data for analysis..."
```
Comment (neisw) [inline ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:420]: Thanks Deep!  Definitely going to want to get feedback from Stephen and possible Devan.  For my own curiosity, is this anything we could do as a test (inside the e2e test execution itself)?   Not saying we should just curious if that is an alternative and if so comparing the approaches.

Comment (neisw) [reply ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:420]: I suppose what I am wondering is if we could just put the calls to get those invariant values in

https://github.com/openshift/origin/blob/748941c870eb4dce2ab6b5c671ababf32a78bb2c/pkg/riskanalysis/write_test_failure_summary.go#L18

We will have to do work in that area one way or another  and if it is less intrusive / more straight forward we might consider it

**Comment by stbenjam** [comment]
Does this replace https://github.com/openshift/release/pull/34834? The comments on there apply here too

**Comment by neisw** [comment]
We don't have access to the gather extra data at the time the test is running.  However we are looking to see if we can collect the invariant data we need within the code that builds the risk analysis data itself and not need this.  But this will be fall back if we can't.

**Comment by neisw** [comment]
Deep I believe https://github.com/openshift/origin/pull/27731#issuecomment-1432408798 will do what I'm looking for.

https://github.com/openshift/origin/blob/3ce40af44d5ce5b01786425ecaacf518b0b51a6c/pkg/synthetictests/platformidentification/types.go#L44 might be worth a review to see if we can use in in other cases / at least look to see if the conversions to 'sippy' values is helpful (possibly in sippy).

**Pull request closed without merging**